frontend/sw360-portlet/src/main/resources/META-INF/resources/html/moderation/view.jsp

```diff
@@ -164,7 +239,18 @@
                             </table>
                         </div>
                         <div id="tab-ClosedMR" class="tab-pane">
-                            <table id="closedModerationsTable" class="table table-bordered"></table>
+                            <table id="closedModerationsTable" class="table table-bordered aligned-top">
+                            <colgroup>
+                                <col />
+                                <col />
+                                <col style="width: 25%;" />
+                                <col style="width: 20%;" />
+                                <col style="width: 20%;" />
+                                <col style="width: 35%;" />
+                                <col />
+                                <col style="width: 1.7rem;" />
+                            </colgroup>
```

Comment:
Is it possible ` <colgroup>` can be removed and handle it while creating Datatable, like passing width parameter similar to line 502 ?
Indentation between Table and colgroup

Reply:
Done